pyuoi/linear_model/elasticnet.py

```diff
@@ -49,20 +51,50 @@ def selection_lm(self):
         return self.__selection_lm
 
     def get_reg_params(self, X, y):
+        """Calculates the regularization parameters (alpha and lambda) to bbe
+        used for the provided data.
+
+        Note that the Elastic Net has two regularization parameters: alpha,
+        which specifies the ratio between the L1 and L2 penalties, and lambda,
+        which specifies the penalty on both regularization terms. We adhere to
+        these variable names, but scikit-learn uses l1_ratio for alpha and
+        alpha for lambda, which can become confusing.
```

Comment:
I would get rid of ", which can become confusing."

Reply:
Resolved in 20d6612.